### PR TITLE
Simplify verification method definition.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #50 by simplifying the definition of "verification method".

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/controller-document/pull/121.html" title="Last updated on Nov 20, 2024, 4:18 PM UTC (32817f4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/controller-document/121/f2baa07...32817f4.html" title="Last updated on Nov 20, 2024, 4:18 PM UTC (32817f4)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -578,10 +578,10 @@ <h3>Terminology</h3>
...
           <dd>
             <p>
-A set of parameters that can be used together with a process to independently
-verify a proof. For example, a cryptographic public key can be used as a
-verification method with respect to a digital signature; in such use, it
-verifies that the signer possessed the associated cryptographic private key.
+A method and its parameters used to independently verify a proof. For
+example, a cryptographic public key can be used as a verification method with
+respect to a digital signature; in such use, it verifies that the signer
+possessed the associated cryptographic private key.
```
Comment (dlongley) [inline index.html:584]: I'd like to keep pushing the community to understand that possession of the key isn't required, only access to an interface that will make use of it. This is a perhaps subtle but important shift in thinking and helps better inform security/threat modeling.

```suggestion
used the associated cryptographic private key.
```

**Review by dlongley — APPROVED** [review]

**Review by David-Chadwick — APPROVED** [review]

**Review by brianorwhatever — APPROVED** [review]

**Review by iherman — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -578,10 +578,10 @@ <h3>Terminology</h3>
           <dt><dfn class="export">verification method</dfn></dt>
           <dd>
             <p>
-A set of parameters that can be used together with a process to independently
-verify a proof. For example, a cryptographic public key can be used as a
-verification method with respect to a digital signature; in such use, it
-verifies that the signer possessed the associated cryptographic private key.
+A method and its parameters used to independently verify a proof. For
```
Comment (TallTed) [inline index.html:581]:
```suggestion
A method and its parameters, used to independently verify a proof. For
```

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-11-20-vcwg#section1-3) on 2024-11-20

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.3. Simplify verification method definition. (pr controller-document#121)

_See github pull request [controller-document#121](https://github.com/w3c/controller-document/pull/121)._




**Brent Zundel:** To address issue #150. PR has nothing but approvals with only a minor proposed change from TallTed.  

**Manu Sporny:** Mike Jones felt we should simplify it and proposed some text.  

**Brent Zundel:** Very simple and straightforward, please review, will likely be merged in three days.  

</details>

**Review by brentzundel — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**